### PR TITLE
Create amazonwarrior.ms

### PR DESCRIPTION
**Pull request by IdkMyNamePleaseTellMe**
proof for jump 3: https://www.youtube.com/watch?v=KQDIkL0kN5M 1:26 (she jumps to Steppen Wolf) and 
https://www.youtube.com/watch?v=UHaRbkc0a3s 0:05
Run 3: https://www.youtube.com/watch?v=L5xoseGkTTA 2:12
Proof for strength: https://www.youtube.com/watch?v=KQDIkL0kN5M 1:55-1:57 (They break heavy wood pillars then catch and hold a thick stone wall)
Proof of health_boost: https://www.youtube.com/watch?v=KQDIkL0kN5M 3:22 (She gets hit off the horse and it does not kill her xD) and https://www.youtube.com/watch?v=KQDIkL0kN5M 0:30-0:35
Proof of armor: https://www.youtube.com/watch?v=KQDIkL0kN5M 0:02
Proof of lasso: https://www.youtube.com/watch?v=KQDIkL0kN5M 2:51
Proof of sword: https://www.youtube.com/watch?v=KQDIkL0kN5M 0:34
Proof of shield: https://www.youtube.com/watch?v=KQDIkL0kN5M 0:22
Proof of Bows: https://www.youtube.com/watch?v=KQDIkL0kN5M 3:05
Proof for fire on bows: https://www.youtube.com/watch?v=Pn6heJ_ET7U 1:19

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/chars/amazonwarrior.ms [hunk 1]
```diff
@@ -0,0 +1,37 @@
+maxHearts: 20
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/amazonwarrior.ms]: these aren't .ms files they are .yml files

File: CommandHelper/LocalPackages/commands/chars/amazonwarrior.ms [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- /effect <plr> 21 9999 4 true
+- /give <plr> 420 1 16:5 name:&6Am&7az&eon&8ia&6n_&eLasso
+- /give <plr> minecraft:iron_sword 1 16:5 name:&6Am&7az&eon&8ia&6n_&eSword
+- /give <plr> minecraft:bow 1 50:1 51:1 name:&6Am&7az&eon&8ia&6n_&eBow
+- /give <plr> minecraft:arrow 10
+- /give <plr> minecraft:shield 1 0 name:&6Am&7az&eon&8ia&6n_&eShield
+- /give <plr> minecraft:iron_helmet 1 0 0:10 1:10 3:10 4:10 7:10 name:&6Am&7az&eon&8ia&6n_&eHe&6lm&7et
+- /give <plr> minecraft:iron_chestplate 1 0 0:10 1:10 3:10 4:10 7:10 name:&6Am&7az&eon&8ia&6n_&eCh&6es&7tpl&8ate
+- /give <plr> minecraft:iron_leggings 1 0 0:10 1:10 3:10 4:10 7:10 name:&6Am&7az&eon&8ia&6n_&eLe&7gg&6in&8gs
+- /give <plr> minecraft:iron_boots 1 0 0:10 1:10 3:10 4:10 7:10 name:&6Am&7az&eon&8ia&6n_&eBo&6o&7ts
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/amazonwarrior.ms]: either they get armor, or they get health & resistance, which do you want?

File: CommandHelper/LocalPackages/commands/chars/amazonwarrior.ms [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+chatcolor: '&6'
+class: amazonwarrior
+haste: 6
+jump: 6
+name: §6§lAm§7§laz§e§lon §8§lWa§6§lrr§elior
+op: false
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/amazonwarrior.ms]: you need to add this somewhere

allowMultiple: true

File: CommandHelper/LocalPackages/commands/chars/amazonwarrior.ms (hunk 2 above)
Comment (IdkMyNamePleaseTellMe) [inline CommandHelper/LocalPackages/commands/chars/amazonwarrior.ms]: I cant edit

Comment (malonnnn) [reply CommandHelper/LocalPackages/commands/chars/amazonwarrior.ms]: There should be no reason why you can't edit, this code is literally under your account, which is why it has ur name by it. what error are you getting?

**Pull request closed without merging**